src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -621,6 +621,12 @@ public static HashSet<string> GetLanguageWorkerListToStartInPlaceholder(this IEn
             return placeholderRuntimeSet;
```

Comment:
just above this (i can't link directly to it) -- can you change the `string.IsNullOrEmpty()` check to check `!IsInProc()` instead?

We don't want that placeholder list to ever include "dotnet"

```
if (!string.IsNullOrEmpty(workerRuntime))
{
    placeholderRuntimeSet.Add(workerRuntime);
}
```